scala/scala.bzl

```diff
@@ -109,7 +132,8 @@ rm -rf {out}_tmp
       arguments=[f.path for f in ctx.files.srcs])
 
 def _compile_or_empty(ctx, jars, buildijar):
-  if len(ctx.files.srcs) == 0:
+  # TODO We assume that if a srcjar is present, that it is not empty... is there a way to check if it has files?
```

Comment:
no, we can't read the file before we do an action. I think we should not support empty srcjar.